dss/config.py

```diff
@@ -541,8 +543,12 @@ def checkout_bucket(self) -> str:
         return self._checkout_bucket_getter()
 
     @property
-    def flashflood_prefix(self) -> str:
-        return os.environ[self._flashflood_prefix_envvar]
+    def flashflood_prefix_read(self) -> str:
+        return os.environ[self._flashflood_prefix_read_envvar]
+
+    @property
+    def flashflood_prefix_write(self) -> typing.Tuple[str, ...]:
+        return tuple(os.environ[self._flashflood_prefix_write_envvar].split(","))
```

Comment:
I realize that you are using them differently, but it's unexplained here why `Replica.flashflood_prefix_write()` would return a `tuple` while `Replica.flashflood_prefix_read()` returns a `str`.  I would consider making them consistent.  If not, a short docstring here explaining it would aid someone like me in the future.

Reply:
docstrings added